optimum/commands/export/openvino.py

```diff
@@ -272,6 +281,7 @@ def run(self):
                     "quant_method": "awq" if self.args.awq else "default",
                     "sensitivity_metric": self.args.sensitivity_metric,
                     "scale_estimation": self.args.scale_estimation,
+                    "dtype": "int" if "int" in self.args.weight_format else self.args.weight_format,
```

Comment:
i think this naming is confusing here as dtype usually takes values like float16, int8. Also "`dtype`" only makes sense when both weights and activations have the same format. Storing `weight_format` and using it for dispatching makes more sense for me. WDYT ?

Reply:
@IlyasMoutawwakil Thanks, I agree your points. I also had an idea to name it `weight_format`, but decided against it.

If we name it `weight_format`, we can either store values in the form of `int8` (similarly to CLI), or in the format as in the PR right now, i.e. `int`. The problem I see with the first approach is that there will be a redundancy as we also have a `bits` field. With the second approach there will be a misaligned between CLI and OV*QuantizationConfig as the same variable name corresponds to values in different format.

That's why I decided to introduce a separate field. Initially if was `data_type`, but I renamed it to `dtype`. Maybe this is not the best option.

I guess there is some drawback to any approach. Which case do you think is less critical?

Reply:
Ideally, I would remove `bits` and `sym` in favor of `weight_format`, because it is more flexible. And also `sym` is not relevant to `mxfp4_e2m1` type. For activation quantization config we can also add `activation_format` field.

But this will require quite a refactoring. Also, support for `mxfp4_e2m1` is kind of in an experimental stage right now and I we don't have to fully commit to it yet.

Reply:
I think the redundancy is alright in this case (without removing `bits` and `sym`).

Reply:
Done